src/index.js

```diff
@@ -1,65 +1,172 @@
+
 /**
- * App ID for the skill
+ * This sample demonstrates a simple skill built with the Amazon Alexa Skills Kit.
+ * The Intent Schema, Custom Slots, and Sample Utterances for this skill, as well as
+ * testing instructions are located at http://amzn.to/1LzFrj6
+ *
+ * For additional samples, visit the Alexa Skills Kit Getting Started guide at
+ * http://amzn.to/1LGWsLG
  */
 
-var APP_ID = "amzn1.echo-sdk-ams.app.23ed8a00-90ef-4ba5-b418-a1d4d8ec5e32";
-
-var AlexaSkill = require('./AlexaSkill');
-
-
+// Route the incoming request based on type (LaunchRequest, IntentRequest,
+// etc.) The JSON body of the request is provided in the event parameter.
 
-var ReverseGeek = function () {
-    AlexaSkill.call(this, APP_ID);
+exports.handler = function (event, context) {
```

Comment:
Good question.  This is provided by AWS Lambda to provide some additional information about the currently running lambda and allow you to signify when your lambda is complete, by calling its `succeed` or `fail` function.  See [docs](http://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-using-old-runtime.html#nodejs-prog-model-oldruntime-context-methods)